WaybackWhen.py

```
@@ -1012,28 +1062,37 @@ def run(self):
                     for cf_info in list(self.crawling_futures_set): # Iterate over a copy to allow modification
                         if cf_info[0] == future:
                             self.crawling_futures_set.remove(cf_info)
+                            if future.cancelled():
+                                log_message("DEBUG", f"Skipping cancelled crawl task.", debug_only=True)
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)